### PR TITLE
UI: Light Theme fix

### PR DESCRIPTION
**Pull request by cannarelladev**
# Description

UI: Light theme (BETA) first release

### CODE REVIEW

File: frontend/src/components/activePage/TableInstance/RowInstanceActions/RowInstanceActionsDefault.tsx [hunk 1]
```diff
@@ -37,6 +37,16 @@ const RowInstanceActionsDefault: FC<IRowInstanceActionsDefaultProps> = ({
...
   };
 
+  const classFromPropsMobile = () => {
+    if (extended) {
+      return 'link';
+    } else {
+      if (status === 'VmiReady' && gui) {
+        return 'success';
+      } else return 'primary';
+    }
```
Comment (QcFe) [inline frontend/src/components/activePage/TableInstance/RowInstanceActions/RowInstanceActionsDefault.tsx:47]:
```suggestion
    if (extended) return 'link';
    if (status === 'VmiReady' && gui) return 'success';
    return 'primary';
```

**Comment by giorio94** [comment]
@cannarelladev Could you please delete old commented code?

**Comment by cannarelladev** [comment]
@giorio94 Done!

**Review by QcFe — APPROVED** [review]

**Pull request merged**